doc/nrf/releases_and_maturity/migration/migration_guide_spm_to_tf-m.rst

```diff
@@ -0,0 +1,59 @@
+.. _spm_to_tfm_migration:
+
+Migrating from Secure Partition Manager to Trusted Firmware-M
+#############################################################
+
+The Nordic Secure Partition Manager (SPM) was replaced with Trusted Firmware-M (TF-M) as the default trusted execution solution in the |NCS| v2.1.0.
+This change enhances the security features of the SDK by integrating the more widely adopted TF-M that aligns with the Arm Platform Security Architecture (PSA).
+
+Porting SPM Secure Services
+***************************
+
+Migration from SPM to TF-M requires changes in the application code and the partition configuration.
+The interface to TF-M is different from the interface to SPM.
+Due to that, the application code that uses the SPM Secure Services needs to be ported to use TF-M instead.
+
+TF-M can replace the following SPM services:
+
+.. list-table:: Migrating SPM Secure Services to TF-M
+   :widths: 40 60
+   :header-rows: 1
+
+   * - SPM Function
+     - TF-M Replacement
+   * - ``spm_request_system_reboot``
+     - ``tfm_platform_system_reset``
+   * - ``spm_request_random_number``
+     - ``psa_generate_random`` or ``entropy_get_entropy``
+   * - ``spm_request_read``
+     - ``tfm_platform_mem_read`` or ``soc_secure_mem_read``
+   * - ``spm_s0_active``
+     - ``tfm_platform_s0_active``
+   * - ``spm_firmware_info``
+     - ``tfm_firmware_info``
+
+
+SPM Secure Services with no replacement
+***************************************
+
+The following SPM services have no replacement in TF-M:
+
+* ``spm_prevalidate_b1_upgrade``
+* ``spm_busy_wait``
+* ``spm_set_ns_fatal_error_handler``
+
```

Comment:
```suggestion
* ``spm_set_ns_fatal_error_handler``

Remove these SPM services from your project.
```
So that we can have the required action stated: https://github.com/nrfconnect/sdk-nrf/pull/19562/files#r1940783342